src/Applications/LDAS_App/CMakeLists.txt

```diff
@@ -16,11 +16,11 @@ ecbuild_add_executable (
 
 ecbuild_add_executable (
    TARGET mwrtm_bin2nc4.x
-   SOURCES mwrtm_bin2nc4.F90
+   SOURCES util/inputs/mwrtm_bin2nc4.F90
    LIBS GEOSlandassim_GridComp)
    
 install(
-   PROGRAMS ldas_setup process_hist.csh process_rst.csh util/average_ensemble_forcing.py
+   PROGRAMS ldas_setup process_hist.csh process_rst.csh ens_forcing/average_ensemble_forcing.py
```

Comment:
CMakelists.txt should need more edits.  Some of the new scripts in ./ens_forcing should also be copied to ./bin.

Reply:
I just pushed updates in ens_forcing/ .  CmakeLists is modified to install the codes to bin/.